test/ui-test/testSuites/suite_status/shared/scripts/bdd_hooks.py

```diff
@@ -16,6 +16,7 @@ def hook(context):
     context.userData = {}
     context.userData["aut_name"] = _status_desktop_app_name
     context.userData["status_data_folder_path"] = _status_data_folder_path
+    context.userData["fixtures_root"] = os.path.join(os.path.dirname(__file__), "../../../fixtures/")
```

Comment:
FYI, we found that this doesn't work for Mac. I updated my PR (where you most likely took inspiration :wink: ) so you can update with that.

Reply:
hehe ok, close save :D